executor/src/witgen/query_processor.rs

```diff
@@ -0,0 +1,94 @@
+use ast::analyzed::{Expression, PolynomialReference};
```

Comment:
is this file just extracted from somewhere else?

Reply:
Yes, from [here](https://github.com/powdr-labs/powdr/blob/1283f61dafb5207c44e7e20257bca961db9bdacc/executor/src/witgen/generator.rs#L310-L377).